docs/usage/routes.md

```diff
@@ -120,4 +120,42 @@ spec:
         termination: reencrypt
         insecureEdgeTerminationPolicy: Redirect
 ```
+### Host URL for Route in Argo CD Status
 
+When setting up access to Argo CD via a Route or an Ingress, one can easily retrieve the URL used for accessing the Argo CD installation through the ArgoCD Operand's `.status` field. To expose the `.host` field, run `kubectl edit argocd argocd` and then edit the Argo CD instance server to have route enabled as `true`, like so: 
+
+```yaml
+server:
+    autoscale:
+      enabled: false
+    grpc:
+      ingress:
+        enabled: false
+    ingress:
+      enabled: false
+    route:
+      enabled: true
+    service:
+      type: ""
+  tls:
+    ca: {}
+```
+If a route is found, you host URL can now be accessed by inspecting your Argo CD instance. It will look like the following: 
+
+```yaml
+status:
+  applicationController: Running
+  dex: Running
+  host: argocd-server-default.my-cluster-url.openshift.com
+  phase: Available
+  redis: Running
+  repo: Running
+  server: Running
+  ssoConfig: Unknown
+```
+
+If the status of `.host` is pending, this will affect the overall status of the Operand by making it `Pending` instead of `Available`. Once the Route is available, the status of the Operand should change to `Available`.
```

Comment:
```suggestion
If the status of the Route is pending, this will affect the overall status of the Operand by making it `Pending` instead of `Available`. Once the Route is available, the status of the Operand should change to `Available`.
```